concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -191,7 +191,7 @@ Hale-1973-1798-189	Human_Body-90	descend	v	02b1.14	2014	GO DOWN (DESCEND)
 Hale-1973-1798-190	Human_Body-91	accompany	v	02b1.15	1542	ACCOMPANY
 Hale-1973-1798-191	Human_Body-92	escort	v	02b1.16		
 Hale-1973-1798-192	Human_Body-93	follow	v	02b1.17	1085	FOLLOW
-Hale-1973-1798-193	Human_Body-94	chase	v	02b1.18		
+Hale-1973-1798-193	Human_Body-94	chase	v	02b1.18	1047	PURSUE
```

Comment:
This should work, since the definition at https://github.com/concepticon/concepticon-data/blob/master/concepticondata/concepticon.tsv#L1048 reads: "Trying to achieve, to catch, or to reach something or someone."

Reply:
@martino-vic The mappings of other glosses to the concept set are more important than the definition. The definition is only secondary. Since there are other _chase_ glosses mapped to the concept, this can be left as is.